src/support/valid-custom-element.js

```diff
@@ -1,3 +1,14 @@
 export default function (name) {
-  return name.indexOf('-') > 0;
+  const reservedNames = [
+    'annotation-xml',
+    'color-profile',
+    'font-face',
+    'font-face-src',
+    'font-face-uri',
+    'font-face-format',
+    'font-face-name',
+    'missing-glyph'
+  ];
+
+  return name.indexOf('-') > 0 && name.toUpperCase() === name && !reservedNames.includes(name);
```

Comment:
`includes` [isn't supported in ie](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes#Browser_compatibility) it seems. Probably better to do a `indexOf() === -1` check.